doc/conf.py

```diff
@@ -0,0 +1,223 @@
+# directories to ignore when looking for source files. This image also affects
+# html_static_path and html_extra_path.
+exclude_patterns = [
+    "_build",
+    "_static/v0.2.0/*.ipynb",
+]
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for a
+# list of builtin themes.
+html_theme = "sphinx_rtd_theme"
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files, so
+# a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = [
+    "_static",
+]
+html_css_files = [
+    "style.css",
+]
+
+# Syntax highlighting
+pygments_style = "friendly"
+
+napoleon_google_docstring = False
+napoleon_use_param = False
+napoleon_use_ivar = True
+
+
+# Logo
+cmap = "hot"
+html_logo = f"_static/icon/logo.svg"
+html_favicon = f"_static/icon/logo.png"
+
+# Read the Docs theme options
+html_theme_options = {
+    "prev_next_buttons_location": "bottom",
+}
+
+# Figure references
+numfig = True
+
+# nbsphinx configuration
+# Taken from nbsphinx' own nbsphinx configuration file, with slight
+# modification to point nbviewer and Binder to the GitHub master links
+# when the documentation is launched from a kikuchipy version with
+# "dev" in the version.
+if "dev" in version:
+    release_version = "master"
+else:
+    release_version = "v" + version
+# This is processed by Jinja2 and inserted before each notebook
+nbsphinx_prolog = (
+    r"""
+{% set docname = 'doc/' + env.doc2path(env.docname, base=None) %}
+.. raw:: html
+    <style>a:hover { text-decoration: underline; }</style>
+    <div class="admonition note">
+      This page was generated from
+      <a class="reference external" href="https://github.com/pyxem/kikuchipy/blob/"""
+    + f"{release_version}"
+    + r"""/{{ docname|e }}">{{ docname|e }}</a>.
+      Interactive online version:
+      <span style="white-space: nowrap;"><a href="https://mybinder.org/v2/gh/pyxem/kikuchipy/"""
+    + f"{release_version}"
+    + r"""?filepath={{ docname|e }}"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a>.</span>
+      <script>
+        if (document.location.host) {
+          $(document.currentScript).replaceWith(
+            '<a class="reference external" ' +
+            'href="https://nbviewer.jupyter.org/url' +
+            (window.location.protocol == 'https:' ? 's/' : '/') +
+            window.location.host +
+            window.location.pathname.slice(0, -4) +
+            'ipynb">View in <em>nbviewer</em></a>.'
+          );
+        }
+      </script>
+    </div>
+.. raw:: latex
+    \nbsphinxstartnotebook{\scriptsize\noindent\strut
+    \textcolor{gray}{The following section was generated from
+    \sphinxcode{\sphinxupquote{\strut {{ docname | escape_latex }}}} \dotfill}}
+"""
+)
+# https://nbsphinx.readthedocs.io/en/0.8.0/never-execute.html
+nbsphinx_execute = "always"  # auto, always, never
```

Comment:
We've switched to "auto" for kikuchipy, since we've started saving the output in two of the most computationally expensive notebooks in the user guide. With "auto", these aren't run by Sphinx. The benefit of "always" is that any breaking change to the notebooks are caught immediately. We've found a middle ground by [running and testing all notebooks once a week](https://github.com/pyxem/kikuchipy/blob/develop/.github/workflows/test_documentation_notebooks.yml) using `nbval`.